### DISCUSSION TITLE
Add command that returns espanso if expansions are enabled/disabled

### DISCUSSION THREAD

**Discussion by pstuifzand** [espanso/espanso#1264] [Feature requests and Ideas]
**Is your feature request related to a problem? Please describe.**

I created an `Argos` menu item in Ubuntu. It would be great to show the expansion status of Espanso (cmd toggle/enable/disable).

**Describe the solution you'd like**

Add a command that returns "enabled" of "disabled". `espanso cmd status`. This is different from `espanso status`, which only checks if espanso is running.

https://github.com/federico-terzi/espanso/blob/e37aa986166b8e55f910afab23cf430a1772e6e9/espanso/src/cli/cmd.rs#L44-L49

**Describe alternatives you've considered**

When the status was toggled, it is possible to grep the status from the espanso logs.

**Additional context**

![espanso item in panel](https://user-images.githubusercontent.com/64524/160102715-45d92b1c-a848-4d43-8acf-74927f73b4b9.png)

Argos - https://github.com/p-e-w/argos

**Comment by federico-terzi** [comment]
@pstuifzand Thank you for the feedback! Good point, I'll add it to the roadmap :)
